realm/options.h

```diff
@@ -0,0 +1,241 @@
+/* wolfssl options.h
```

Comment:
Is this file actually used for anything?

Reply:
options.h is included so it can be used for non-autotools configure builds.